src/Swoole/SwooleClient.php

```diff
@@ -19,12 +19,6 @@
 
 class SwooleClient implements Client, ServesStaticFiles
 {
-    const STATUS_CODE_REASONS = [
-        419 => 'Page Expired',
-        431 => 'Request Header Fields Too Large',                             // RFC6585
-        451 => 'Unavailable For Legal Reasons',                               // RFC7725
-    ];
-
```

Comment:
Removing this here is a breaking change.

Reply:
@driesvints I made a small change to fix this